federation-js/src/composition/validate/postComposition/__tests__/executableDirectivesInAllServices.test.ts

```diff
@@ -83,6 +85,12 @@ describe('executableDirectivesInAllServices', () => {
       Array [
         Object {
           "code": "EXECUTABLE_DIRECTIVES_IN_ALL_SERVICES",
+          "locations": Array [
```

Comment:
Unsure if `locations` makes sense here at all. In this case, `serviceB/C` are the ones missing the implementation, but the error points back to the implementation in `serviceA`. It's probably the _most useful_ location if there is one, just not sure that it is given the nature of the error.

Reply:
that makes sense, would it be okay for this error in this case to just not have location? I will delete the node since I do think so, but making sure.

Reply:
got it. I do think it would be helpful for someone to see some UI on that part of the code saying "hey you need to implement this everywhere". Could this be another place that we comment in [preparation for adding service name as structured data](https://github.com/apollographql/federation/issues/705)? In the future, we could specify "hey this is pointing to this line in service A".

I don't know your teams method of tracking TODOs - I will also definitely add these to the open issue.

Reply:
Yep that sounds good to me!